src/Types.ts

```diff
@@ -49,6 +50,8 @@ export type TaskStatus = 0 | 1 | 2 | 3;
 
 export type CreatedTaskStatus = 0 | 1 | 2;
 
+export type ApprovalStatus = 1 | 2 | 3;
```

Comment:
Can these start at 0, just like the CreatedTaskStatuses?

Reply:
@bzzbbz correct me if I misunderstood, but I think that this is intentionally `1 | 2 | 3` so that the status states match up between requester and tasker:

Tasker: In progress (0), Pending verification (1), Verified & paid (2), Abandoned (3)

Requester: Not reviewed (1), Approved (2), Rejected (2)

Status of 0 isn't relevant for requesters.

Reply:
It's done intentionally to match the "status" column in the TaskUsers table in the database. Status of 0 in TaskUsers means that the user has started the task, but not completed it, so these tasks are filtered out of the approval table. Status 1 means that the task if completed, but no one had viewed it yet.